lsif-java/src/main/scala/com/sourcegraph/lsif_java/commands/SnapshotLsifCommand.scala

```diff
@@ -174,14 +176,20 @@ object SnapshotLsifCommand {
         .map(m => m.getIdentifier -> m)
         .toMap
     val hovers: Map[String, String] = monikers.map { case (sym, obj) =>
-      val hoverMessage =
-        for {
-          resultSet <- monikerInverse.get(obj.getId).toList
-          hoverEdge <- hoverEdges.get(resultSet).toList
-          hoverResult <- hoverVertexes.get(hoverEdge).toList
-          contents <- hoverResult.getContentsList.asScala
-        } yield contents.getValue.trim
-      sym -> hoverMessage.mkString("\n\n")
+      val hover =
```

Comment:
Nit: for comprehensions become hard to read when you add a complicated chain of operations on top of it. I think this diff would read cleaner by keeping the original for comprehension and adding a separate statement instead